src/features/movies/MovieList/index.js

```diff
@@ -14,17 +14,21 @@ export const MovieList = () => {
         dispatch(fetchMovies());
     }, [dispatch]);
 
+    const formatDate = (date) => {
+        return date.substr(0, 4);
+    };
+
     switch (fetchStatus) {
         case "completed":
-            const moviesData = movies.data
+            const moviesData = movies.data;
             return (
                 <Content>
                     <Title>Popular movies</Title>
                     <Wrapper>
                         {moviesData.results.map((movie) =>
                             <MovieTile key={movie.id}
                                 title={movie.title}
-                                year={"2020"}
+                                year={formatDate(movie.release_date)}
```

Comment:
Yes, it is working, however substr() method is deprecated - this means it is working now but is no longer recommended and could stop working at any time. Also you dont have to create another function, instead you could go with 
`year={movie.release_date.substr(0, 4)}`

[substr() documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr)

Reply:
i thinking about slice 😅, but i couldnt remember this and i found substr 😅, we dont need function for this you right

Reply:
I don't know if it makes sense to make an object and write unnecessary lines of code, because we always get the same string with the year in the first 4 positions

Reply:
i change to slice without exttra function